iml-gui/crate/src/page/target_dashboard.rs

```diff
@@ -0,0 +1,69 @@
+use crate::Msg;
+use iml_wire_types::warp_drive::ArcCache;
+use seed::{prelude::*, *};
+use std::fmt;
+
+#[derive(Default)]
+pub struct Model {
+    pub target_name: String,
+}
+
+pub enum TargetUsage {
+    Object,
+    File,
+}
+
+impl fmt::Display for TargetUsage {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match *self {
+            TargetUsage::Object => write!(f, "Object"),
+            TargetUsage::File => write!(f, "File"),
+        }
+    }
+}
+
+impl From<&str> for TargetUsage {
+    fn from(item: &str) -> Self {
+        if item.contains("OST") {
+            TargetUsage::Object
+        } else {
+            TargetUsage::File
+        }
+    }
+}
+
+pub enum TargetDashboard {
+    MdtDashboard,
+    OstDashboard,
+}
+
+impl fmt::Display for TargetDashboard {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match *self {
+            TargetDashboard::OstDashboard => write!(f, "nQGYg-QWz/target-ost-dashboard"),
+            TargetDashboard::MdtDashboard => write!(f, "Fl_8QJQZk/target-mdt-dashboard"),
+        }
+    }
+}
+
+impl From<&str> for TargetDashboard {
+    fn from(item: &str) -> Self {
+        if item.contains("OST") {
+            TargetDashboard::OstDashboard
+        } else {
+            TargetDashboard::MdtDashboard
+        }
+    }
+}
+
+pub fn view(_: &ArcCache, model: &Model) -> impl View<Msg> {
+    let object_or_files_title: TargetUsage = (model.target_name.as_str()).into();
+    let dashboard_key: TargetDashboard = (model.target_name.as_str()).into();
+
+    iframe![attrs! {
+        At::Src => format!("https://localhost:7444/grafana/d/{}?var-target_name={}&var-object_or_files_title={}&orgId=1&from=now-15m&to=now&kiosk", dashboard_key, &model.target_name, object_or_files_title),
```

Comment:
```suggestion
        At::Src => format!("/grafana/d/{}?var-target_name={}&var-object_or_files_title={}&orgId=1&from=now-15m&to=now&kiosk", dashboard_key, &model.target_name, object_or_files_title),
```

We can use webpack proxy to avoid hardcoding urls.

ref:

https://github.com/whamcloud/integrated-manager-for-lustre/blob/5a0d06eeeb6e20024d4b560af964043b4629d5ae/iml-gui/configs/webpack.config.js#L39